src/components/shared/forms/DateRange.vue

```diff
@@ -195,10 +196,27 @@
             },
 
             quickSet(period) {
-                if (period === 'today') this.isRange = false
-                else this.isRange = true
-                this.localStart = moment().add(-1, period)
-                this.localEnd = moment()
+                if (period === 'today') {
+                    this.isRange = false
+                    this.localStart = moment().add(-1, period)
+                    this.localEnd = moment()
```

Comment:
Should this be the end on of today rather than now?

Reply:
Maybe all of these should be normalised to use the start and end of the day before it applies any period logic.